tests/editAndClear.test.js

```diff
@@ -0,0 +1,179 @@
+/*
+ * @jest-environment jsdom
+ */
+import Collection from '../src/modules/Collection.js';
+import Render from '../src/modules/Render.js';
+
+describe('update and clear', () => {
+  /*
+  * update description property
+  */
+  test('update description', () => {
+    const collection = new Collection('tasks', [
+      {
+        description: 'first',
+        completed: false,
+        index: 1,
+      },
+      {
+        description: 'second',
+        completed: false,
+        index: 2,
+      },
+    ]);
+
+    expect(
+      collection.updateItem(2, 'description', 'test description'),
+    ).toStrictEqual([
+      {
+        description: 'first',
+        completed: false,
+        index: 1,
+      },
+      {
+        description: 'test description',
```

Comment:
- [ ] excellent work, but you have to include a test for the editing task description

Reply:
Hi @VICTOR-CHUKWU, thanks for your comments. In fact, we are including the test for editing (update) the task description. This test starts in line 11 and ends in line 39. We are testing the description update from "second" to "test description". Could you check again, please?